### PR TITLE
Fixed #25728 -- Fixed description of `GEOSGeometry.contains`.

### PR DESCRIPTION
**Pull request by sir-sigurd**
https://code.djangoproject.com/ticket/25728

### CODE REVIEW

File: docs/ref/contrib/gis/geos.txt [hunk 1]
```diff
@@ -346,7 +346,7 @@ return a boolean.
 
 .. method:: GEOSGeometry.contains(other)
 
-Returns ``True`` if :meth:`GEOSGeometry.within` is ``False``.
+Returns ``True`` if and only if other.\ :meth:`within <GEOSGeometry.within>`\(this) is ``True``.
```
Comment (claudep) [inline docs/ref/contrib/gis/geos.txt]: Sorry, but what are those backslashes?

Comment (charettes) [inline docs/ref/contrib/gis/geos.txt]: Not sure about the wording but we fold at 79 characters in docs.

Comment (timgraham) [inline docs/ref/contrib/gis/geos.txt]: I'm also stumped about the markup. I think this is sufficient?
```
Returns ``True`` if and only if :meth:`~GEOSGeometry.within` is ``True``.
```

Comment (sir-sigurd) [inline docs/ref/contrib/gis/geos.txt]: @timgraham 
GEOS docs says:
```
virtual bool 	contains (const Geometry *g) const
 	Returns true if other.within(this) returns true.
```

**Comment by timgraham** [comment]
How about:
```
Returns ``True`` if and only if :meth:`other.within(this) <GEOSGeometry.within>`
returns ``True``.
```
As it stands now, the rendered version is a bit confusing since "other" and "(this)" aren't "code-block formatted". I'll push that version if you agree.

**Comment by sir-sigurd** [comment]
I just wanted to make only word `within` a reference, but your version is OK.

**Comment by timgraham** [comment]
merged in 34669ea6b88cefb6e6f7176c71b7947234fa50b4, thanks!

**Pull request closed without merging**